experiments/standalone/Vegetation/timestep_test.jl

```diff
@@ -0,0 +1,249 @@
+import SciMLBase
```

Comment:
It would be nice to have a comment at the top of this file explaining the goal of this experiment - are we just testing running the vegetation model for a single timestep? Are we testing for stability over n days? etc